gravitee-node-vertx/src/main/java/io/gravitee/node/vertx/server/http/VertxHttpServerFactory.java

```diff
@@ -30,18 +33,32 @@
  */
 public class VertxHttpServerFactory extends AbstractVertxServerFactory implements ServerFactory<VertxHttpServer, VertxHttpServerOptions> {
 
+    /**
+     * Constructor without CRL support for backward compatibility.
+     * CRL validation is not supported when using this constructor. Configuring CRL options will result in a startup failure.
+     */
     public VertxHttpServerFactory(
         Vertx vertx,
         KeyStoreLoaderFactoryRegistry<KeyStoreLoaderOptions> keyStoreLoaderFactoryRegistry,
         KeyStoreLoaderFactoryRegistry<TrustStoreLoaderOptions> trustStoreLoaderFactoryRegistry
     ) {
-        super(vertx, keyStoreLoaderFactoryRegistry, trustStoreLoaderFactoryRegistry);
+        this(vertx, keyStoreLoaderFactoryRegistry, trustStoreLoaderFactoryRegistry, new DefaultCRLLoaderFactoryRegistry());
+    }
+
+    public VertxHttpServerFactory(
+        Vertx vertx,
+        KeyStoreLoaderFactoryRegistry<KeyStoreLoaderOptions> keyStoreLoaderFactoryRegistry,
+        KeyStoreLoaderFactoryRegistry<TrustStoreLoaderOptions> trustStoreLoaderFactoryRegistry,
+        CRLLoaderFactoryRegistry crlLoaderFactoryRegistry
+    ) {
+        super(vertx, keyStoreLoaderFactoryRegistry, trustStoreLoaderFactoryRegistry, crlLoaderFactoryRegistry);
```

Comment:
I remember we discussed it, but looking at it with a clear head, I believe we should move it to `NodeCertificatesConfiguration` and follow the way it is done for other registries. WDYT?